MatrixSDK/Space/MXSpace.swift

```diff
@@ -57,6 +57,8 @@ public class MXSpace: NSObject {
     public private(set) var childRoomIds: [String] = []
     public private(set) var otherMembersId: [String] = []
     public private(set) var membersId: [String] = []
+    
+    public var lastSpaceChildrenSummary: MXSpaceChildrenSummary?
```

Comment:
Can we add a comment for the purpose of this property?

Reply:
This member is not needed anymore.